openlibrary/plugins/upstream/utils.py

```diff
@@ -584,13 +584,19 @@ def _get_recent_changes2():
     
     q = {"bot": False, "limit": 100}
     changes = web.ctx.site.recentchanges(q)
-    
+
+    def is_ignored(c):
+        return (
+            # Gio: c.kind=='update' allow us to ignore update recent changes on people
```

Comment:
Delete the names in this comment and the one below.  They don't add anything and are confusing as to whether the name represents the author or the target of the comment.